tests/Utils/ReadsFixture.py

```diff
@@ -29,4 +29,8 @@ def read_json_fixture(fixture: str, base_path: str | None = None):
         """
         Reads a JSON fixture file and returns its contents as a dictionary.
         """
+        if base_path is None:
+            caller_file = stack()[1].filename
+            base_path = Path(caller_file).parent
```

Comment:
`read_raw_json_fixture` wasn't resolving `base_path` correctly when called from `read_json_fixture` as it's based on the caller file path.